introduction.md

```diff
@@ -3,5 +3,5 @@ https://www.codeschool.com/learn/javascript
 *Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?